src/App.js

```diff
@@ -1,11 +1,11 @@
 import React, {Component} from "react";
-import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
 
 import logo from "./logo.svg";
```

Comment:
Since you removed the logo related CSS, please remove this and the logo itself?
Also, we might lose the favicon altogether with these changes. If we do, create an issue to add a favicon.

Reply:
Added an issue https://github.com/s-maheshbabu/silent-alexa/issues/16